app/controllers/doctors_controller.rb

```diff
@@ -17,7 +17,19 @@ def create
   end
 
   def update
-  end
+    @doctor = Doctor.find(params[:id])
+
+      if  @doctor.update(doctor_params)
+       redirect_to  edit_company_registration_path
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 1) spaces for indentation.<br>Put one space between the method name and the first argument.